lib/routers/establishment/rops-overview.js

```diff
@@ -0,0 +1,69 @@
+const { Router } = require('express');
+const { permissions } = require('../../middleware');
+const { BadRequestError } = require('../../errors');
+
+const app = Router({ mergeParams: true });
+
+app.use(permissions('establishment.rops'));
+
+app.get('/overview', (req, res, next) => {
+  const { Project } = req.models;
+  const { ropsYear, establishmentId } = req.query;
+
+  if (!ropsYear) {
+    throw new BadRequestError('ropsYear must be provided');
+  }
+
+  const ropsDueQuery = Project.query()
+    .count()
+    .whereHasAvailability(establishmentId)
```

Comment:
ahh, in that case I'm going to have to make https://github.com/UKHomeOffice/asl-schema/blob/master/schema/project.js#L282 conditional as well.

Reply:
The easiest (although not necessarily most elegant) way to do that would be to add a `.where({ establishmentId })` in here - https://github.com/UKHomeOffice/asl-schema/blob/master/schema/project.js#L300

Reply:
Yup, I've gone a bit further than that to avoid redundant bits in the query: https://github.com/UKHomeOffice/asl-schema/pull/473 just waiting for drone.